src/Routing/RouteBuilder.php

```diff
@@ -388,7 +388,7 @@ public function namePrefix($value = null)
      */
     public function resources($name, $options = [], $callback = null)
     {
-        if (is_callable($options) && $callback === null) {
```

Comment:
why?

Reply:
Should actually be the other way around
Only check callable options if callback is null (not provided), for nano opt.